third_party/blink/renderer/modules/cobalt/h5vcc_experiments/h_5_vcc_experiments.idl

```diff
@@ -0,0 +1,45 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+enum OverrideState {
+  "OVERRIDE_USE_DEFAULT",
+  "OVERRIDE_DISABLE_FEATURE",
+  "OVERRIDE_ENABLE_FEATURE",
+};
+
+dictionary ExperimentConfiguration {
+  record<DOMString, boolean> features;
+  record<DOMString, (DOMString or long or boolean)> featureParams;
+  sequence<unsigned long> experimentIds;
+};
+
+[
+    Exposed=Window,
+    SecureContext
+]
+interface H5vccExperiments {
+  [CallWith=ScriptState, RaisesException]
+  Promise<void> setExperimentState(ExperimentConfiguration experiment_configuration);
+
+  // Instructs Cobalt to delete its current cached experiment configuration. Returns
+  // a promise that resolves/rejects in the event of success/failure.
+  [CallWith=ScriptState, RaisesException]
+  Promise<void> resetExperimentState();
+
+  // The following APIs fetch Cobalt's current value for the given feature name or
+  // feature param.
+  OverrideState getFeature(DOMString feature_name);
+
+  DOMString getFeatureParam(DOMString feature_param_name);
```

Comment:
Can you add a getter attribute for experiment IDs too? `  readonly attribute FrozenArray<unsigned long> activeExperimentIds;`